sdk/lib/pathcch/pathcch.c

```diff
@@ -403,7 +403,7 @@ HRESULT
 APIENTRY
 PathCchAddBackslash(
     _Inout_updates_(cchPath) PWSTR path,
-    _In_ size_t size)
+    _In_ SIZE_T size)
```

Comment:
Please revert, those are `size_t` in small case (and since this is inside the `#ifdef __REACTOS__` part, let's keep it correct).